RAG/chroma_embedding.py

```diff
@@ -0,0 +1,234 @@
+import os
+import json
+import dotenv
+import argparse
+from langchain.embeddings import HuggingFaceEmbeddings
+from langchain.document_loaders import JSONLoader
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain.embeddings import OpenAIEmbeddings
+from langchain.vectorstores import Chroma
+
+class ChromaEmbedding():
```

Comment:
Again, perhaps a base embeddings class, so that if we add new techniques in the future, it can extend from these classes